homeworks/roman.pavliuk_RomanPvlk/5-logic/style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.heading-primary {
+  font-weight: 700;
+  font-size: 48px;
+  color: #221F1F;
+}
+
+.title {
+  margin-top: 122px;
+  margin-bottom: 25px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.field {
```

Comment:
The UI elements should not jump when an user interact with them
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/9177cd9c-c770-4311-b3eb-235921d8f152)
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/47643f36-4775-467d-9269-c1793dc9e70f)